src/prommis/evaporation_pond/evaporation_pond.py

```diff
@@ -0,0 +1,605 @@
+                f"{len(self.config.property_package.phase_list)} phases."
+            )
+        phase_name = self.config.property_package.phase_list.at(1)
+
+        # Build property blocks
+        self.properties_in = self.config.property_package.build_state_block(
+            self.flowsheet().time,
+            defined_state=True,
+            **self.config.property_package_args,
+        )
+        self.properties_out = self.config.property_package.build_state_block(
+            self.flowsheet().time,
+            defined_state=False,
+            **self.config.property_package_args,
+        )
+        self.reactions = self.config.reaction_package.build_reaction_block(
+            self.flowsheet().time,
+            state_block=self.properties_out,
+            **self.config.reaction_package_args,
+        )
+
+        # Construct Ports
+        self.add_port("inlet", self.properties_in)
+        self.add_port("outlet", self.properties_out)
+
+        # Get indexing sets
+        time = self.flowsheet().time
+        comp_set = self.config.property_package.component_list
+        pc_set = self.config.property_package.get_phase_component_set()
+        rxn_basis = self.reactions[time.first()].get_reaction_rate_basis()
+
+        # Get units of measurement
+        flow_basis = self.properties_in[
+            self.flowsheet().time.first()
+        ].get_material_flow_basis()
+        uom = self.config.property_package.get_metadata().derived_units
+
+        if flow_basis is MaterialFlowBasis.molar:
+            mb_units = uom.FLOW_MOLE
+        elif flow_basis is MaterialFlowBasis.mass:
+            mb_units = uom.FLOW_MASS
+        else:
+            raise PropertyPackageError(
+                f"{self.name} - Property package uses a flow basis of 'other'. "
+                "EvaporationPond model only supports mass or molar bases."
+            )
+
+        if rxn_basis is MaterialFlowBasis.molar:
+            rxn_units = uom.FLOW_MOLE
+        elif rxn_basis is MaterialFlowBasis.mass:
+            rxn_units = uom.FLOW_MASS
+        else:
+            raise PropertyPackageError(
+                f"{self.name} - Reaction package uses a reaction basis other than mass or "
+                "mole. EvaporationPond model only supports mass or molar bases."
+            )
+
+        # Build unit model level variables
+        self.surface_area = Var(
+            time,
+            initialize=1000,
+            bounds=(1, None),
+            units=uom.AREA,
+            doc="Surface area of evaporation pond",
+        )
+        self.average_pond_depth = Var(
+            time,
+            initialize=1,
+            bounds=(1e-6, None),
+            units=uom.LENGTH,
+            doc="Average depth of evaporation pond",
+        )
+        self.volume = Var(
+            time,
+            initialize=1000,
+            bounds=(1e-6, None),
+            units=uom.VOLUME,
+            doc="Volume of liquid in pond",
+        )
+
```

Comment:
Is it just a convention to have the evaporation rate as a positive rate, while water loss and precipitation are negative rates? All three involve species leaving the bulk liquid phase.

Reply:
Yes - evaporation rate is usually defined as a positive value, but for material balances we define losses as -ve.